recordedit/form.controller.js

```diff
@@ -471,5 +471,108 @@
         function blurElement(e) {
             e.currentTarget.blur();
         }
+
```

Comment:
Could you put a comment here noting that below is code that is used for scrolling properly on the form create view.

Something like:
`/* ============================*/`
`// below is code for fixing the column names when scrolling